FrontEnd/src/App.js

```diff
@@ -11,26 +13,51 @@ import TermsAndConditions from './components/terms-and-conditions-app/terms_cond
 import ProfilePage from './components/ProfilePage/ProfilePage';
 import MainPage from './components/landing-page/MainPage';
 import ProfileView from "./components/ProfileView/ProfileView";
+import { AuthContext } from "./context";
 
 
 function App() {
+    const [isAuth, setIsAuth] = useState(false);
+    const [isLoading, setLoading] = useState(true);
+
+    useEffect(() => {
+        if (localStorage.getItem("Token")) {
+            setIsAuth(true)
+        }
+        setLoading(false);
+    }, [])
+
+    axios.interceptors.request.use(config => {
```

Comment:
We haven't functionality to refresh token. User just will be logged out (token will be deleted from db) in two weeks since his login time. So I didn't add  a response interceptor.

Reply:
Are you sure the user will be logged out if the local storage value won't change? We may handle 401 error and logout user when receive it

Reply:
Done